### ISSUE TITLE
Float type not compiling properly, missing crate?

### ISSUE THREAD

**Issue by yoroshikun** [Brendonovich/prisma-client-rust#51]
This issue is kinda split into two
1. Float seems to be using a external crate, this possibly needs to be documented as its not included with the crate
2. Even with the crate it seems that from_f64 is not working, is it meant to be this one? [Here](https://github.com/akubera/bigdecimal-rs)

```rust
                Self::ChanceNot(value) => (
                    "chance".to_string(),
                    SerializedWhereValue::Object(vec![(
                        "not".to_string(),
                        PrismaValue::Float(bigdecimal::from_f64(value).unwrap().normalized()) // <-- This bigdecimal
                            .into(),
                    )]),
                ),
```

**Comment by Brendonovich** [comment]
Whoops, I forgot to make `primsa_client_rust` re-export `bigdecimal`. Would you mind using `rev = 7b103ca7e422d2d5a8935818407fc0023f1b3ab8` instead of `tag = "0.4.1"` to try out the fix I pushed? It should also fix #50

**Comment by yoroshikun** [comment]
thanks for the fast reply testing them now!

This issue has been resolved by that commit, however #50  still persists. Will post more info there
